Documentation/ceph-client-crd.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Client CRD
+indent: true
+---
```

Comment:
You need `weight: 3500` so it will show up in the table of contents in the right place. This would put it after the file system CRD topic.

Reply:
fixed